integration/src/test/scala/org/knora/webapi/slice/lists/ListsV2E2Spec.scala

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.lists
+import zio.test.*
+
+import org.knora.webapi.E2EZSpec
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
+import org.knora.webapi.slice.admin.domain.model.ListProperties.ListIri
+
+object ListsV2E2Spec extends E2EZSpec {
+
+  private def v2listsListIri(iri: ListIri) = s"/v2/lists/${urlEncode(iri.value)}"
+  private def v2nodeListIri(iri: ListIri)  = s"/v2/node/${urlEncode(iri.value)}"
+
+  private val knownRootNode = ListIri.unsafeFrom("http://rdfh.ch/lists/0001/notUsedList")
+  private val knownSubNode  = ListIri.unsafeFrom("http://rdfh.ch/lists/0001/notUsedList01")
+
+  override def rdfDataObjects: List[RdfDataObject] =
+    List(
+      RdfDataObject(
+        path = "test_data/project_data/anything-data.ttl",
+        name = "http://www.knora.org/data/0001/anything",
+      ),
+    )
+
+  override def e2eSpec: Spec[env, Any] = suite("the lists API v2 for")(
+    suite("/v2/lists/:listIri should")(
+      test("return 404 for an unknown list") {
+        for {
+          response <- sendGetRequest(v2listsListIri(ListIri.unsafeFrom("http://rdfh.ch/lists/0001/unknown")))
+        } yield assertTrue(response.status.code == 404)
+      },
+      test("return 404 for an existing sub node") {
+        for {
+          response <- sendGetRequest(v2listsListIri(knownSubNode))
+        } yield assertTrue(response.status.code == 404)
+      },
+      test("return a known list") {
+        for {
+          response <- sendGetRequest(v2listsListIri(knownRootNode))
+          bodyStr  <- response.body.asString
+        } yield {
+          assertTrue(
+            response.status.code == 200,
+            bodyStr == """{
```

Comment:
💡 Stuff like this eventually is better serialized on disk with some functionality to capture, write and override (if necessary) through code. i.e. golden tests/characterization tests.